src/rust/engine/process_execution/src/remote_cache.rs

```diff
@@ -470,24 +469,14 @@ impl crate::CommandRunner for CommandRunner {
     let result = if self.cache_read {
       // A future to read from the cache and log the results accordingly.
       let cache_read_future = async {
-        let response = with_workunit(
-          context.workunit_store.clone(),
-          "check_action_cache".to_owned(),
-          WorkunitMetadata {
-            level: Level::Trace,
-            desc: Some(format!("check action cache for {:?}", action_digest)),
-            ..WorkunitMetadata::default()
-          },
```

Comment:
This work unit was moved to `remote.rs` in `check_action_cache`, which is where all the counters are incremented